lec01-introduction.Rmd

```diff
@@ -2,3 +2,171 @@
+- So, if I read in the iris data we were working with previously, we can view it in the console by typing `iris`, just like before, or we can also view the `iris` variable in our environment tab, which shows it to us in a spreadsheet or table format.
+- This panel shows the files in the current directory, any plots we might make later, and also the documentation, which we accessed earlier by typing `?` in front of a function. Here, the documentation is formatted in a way that is easier to read and also provides links to the related sections.
+- Finally we have the text editor panel. This is where we can write scripts, i.e. putting several commands of code together and saving them as a text document so that they are accessible for later and so that we can execute them all at once by running the script instead of typing them in one by one.
+
+So if I create a script with some of the commands you saw earlier, I can save them all as a text file and then run the entire script by clicking the run button, which executes commands in the console. We will use this text editor to explore how to make plots in R.
+
+## Plots
+In R, it is straightforward to make explore your data by creating nice looking plots. E.g. to make a scatter plot, we first tell R that we want to make a plot of the `iris` data frame, and we want the aesthetics of this plot to be that `sepal_length` is on the x axis and `sepal_width` is on the y-axis. Finally, we specify that we want this to be a plot of points, i.e. a scatter plot.
+
+```{r}
+ggplot(iris, aes(x=sepal_length, y=sepal_width)) +
+    geom_point()
+```
+
+This plots shows up in the plot panel in RStudio, which is convenient instead of having it show up as a separate window. You can already see that there seems to be at least some structure to the data. There is one cluster of points up the left corner and at least one in the lower right corner. Hmmmm, I wonder if these correspond to the different species of flowers... Let's find out, by coloring the points according to which species they are.
+
+```{r}
+ggplot(iris, aes(x=sepal_length, y=sepal_width, color=species)) +
+    geom_point()
+```
+
+Likewise, it would be really easy to make a histogram of the petal_length and color it according to the species.
+
+```{r}
+ggplot(iris, aes(x=petal_length, fill=species)) +
+    geom_histogram()
+```
+
+And we could get fancier by simply instructing R to split this plot in vertically stacked facets, or subplots, with one species in each facet.
+
+```{r}
+ggplot(iris, aes(x=petal_length, fill=species)) +
+    geom_histogram() +
+    facet_wrap(~ species, dir='v')
+```
+
+One last example If you don't have a clear idea of what you are looking for in a data set, maybe you are exploring publicly available data that you have not generated yourself, then it can be useful to visualize the relationships between all the variables to get an overview of the data.
```

Comment:
The whole "maybe ..." clause should be in parentheses or something; it's not really related to the rest of the sentence.

Reply:
Can I not use commas instead of parenthesis there? As "parenthetical punctuation" [as it apparently is called](http://www.grammar-monster.com/lessons/commas_for_parenthesis.htm). Is it too unrelated for that?

Reply:
But you can see that most (or even all) of those examples are very short parentheticals. An em-dash would work here too.